### PR TITLE
export default and stateless function snippets

### PR DESCRIPTION
**Pull request by btcalls**
* Added snippet for `export default`
* Added snippet for creating stateless functions

### CODE REVIEW

File: snippets/snippets.cson [hunk 1]
```diff
@@ -159,90 +177,100 @@
...
+      
+      
+      export default ${1:${TM_FILENAME/(.+)\..+|.*/$1/:MyComponent}};$0
     """
-  'flex: 1':
-    'prefix': 'f1'
-    'body': 'flex: ${1:1},$0'
-  'Create StyleSheet':
-    'prefix': 'css'
+  'alignItems: 'center',':
```
Comment (shrugs) [inline snippets/snippets.cson]: atomizr has a bug, please escape these single quotes with a `\`before them

File: snippets/snippets.cson [hunk 2]
```diff
@@ -159,90 +177,100 @@
...
+      <ActivityIndicatorIOS
+        ${1:animating={true\}}
+        ${2:color={\'#808080\'\}}
+        ${3:size={\'small\'\}}
+      />
+      ${4}$0
     """
-  'componentDidUpdate(object prevProps, object prevState)':
-    'prefix': 'cdud'
+  'justifyContent: 'center',':
```
Comment (shrugs) [inline snippets/snippets.cson]: likewise here

File: snippets/snippets.cson [hunk 3]
```diff
@@ -1,119 +1,118 @@
 # Generated with Atomizr - https://github.com/idleberg/atomizr.rb
-'.source.js,source.jsx':
-  'componentWillMount()':
-    'prefix': 'cwm'
-    'body': """
-      componentWillMount() {
-        ${1}
-      }$0
-    """
+'.source.jssource.jsx':
```
Comment (shrugs) [inline snippets/snippets.cson]: add a comma here so it looks like `.source.js,source.jsx`

**Review by shrugs — CHANGES_REQUESTED** [review]

**Comment by shrugs** [comment]
thanks for the PR! Just making sure the atom spec is kept up to date

File: snippets/snippets.cson (hunk 3 above)
Comment (btcalls) [reply snippets/snippets.cson]: Will update, thanks

**Review by shrugs — APPROVED** [review]

**Pull request merged**